include/UIKit/UIApplicationDelegate.h

```diff
@@ -38,10 +38,12 @@
  * App developers should import the following files to use methods requiring projected classes
  * #import <UWP/WindowsFoundation.h>
  * #import <UWP/WindowsMediaSpeechRecognition.h>
+ * #import <UWP/WindowsApplicationModelActivation.h>
 */
```

Comment:
this is probably what you were thinking of @aballway and this has an todo so we are at least tracking the work here. It likely is due to circular deps between Projections and Foundation/Other frameworks creating the issue which I'd love us to solve at some point.